shared/fixes/bash/bootloader_audit_argument.sh

```diff
@@ -1,14 +0,0 @@
-# platform = multi_platform_rhel
-
-{{% if product == "rhel7" %}}
-# Correct the form of default kernel command line in /etc/default/grub
-grep -q ^GRUB_CMDLINE_LINUX=\".*audit=0.*\" /etc/default/grub && \
-  sed -i "s/audit=[^[:space:]\+]/audit=1/g" /etc/default/grub
-if ! [ $? -eq 0 ]; then
-  sed -i "s/\(GRUB_CMDLINE_LINUX=\)\"\(.*\)\"/\1\"\2 audit=1\"/" /etc/default/grub
-fi
-{{% endif %}}
```

Comment:
How about moving `rhel7` section of the fix be moved to the `grub2` [rule](https://github.com/ComplianceAsCode/content/tree/d7df03b5ffb8d76973eab64188bcd41f7835b70f/linux_os/guide/system/auditing/grub2_audit_argument)?

Reply:
Nevermind, I see the other PR now.

Reply:
Yes. However I wonder that the RHEL 6 version does not edit the config file at all.